R/serializer.R

```diff
@@ -652,8 +652,8 @@ createGraphicsDevicePromiseDomain <- function(which = dev.cur()) {
       force(onFulfilled)
```

Comment:
Up above add `stopifnot(which >= 2)` (to make the rest of this easier to reason about)